packages/react-native-reanimated/src/Animated.ts

```diff
@@ -27,6 +27,7 @@ export { ReanimatedFlatList as FlatList } from './component/FlatList';
 export {
   addWhitelistedNativeProps,
   addWhitelistedUIProps,
+  configureLogger,
```

Comment:
Please don't add stuff to `Animated` 😭

Reply:
Mhm. I thought it's similar to calling `Animated.addWhitelistedNativeProps` in terms of behavior. In both cases, you change the configuration of something so I wanted to keep a similar approach.

I can export it as a standalone function renamed to something more meaningful like `configureReanimatedLogger` or something similar if you prefer.

Reply:
The Animated namespace was added to make the migration from React Native's Animated API to Reanimated API. At the moment it's (in my opinion) unnecessary and I'd like to see it removed in Reanimated 4 😅